sample-input/pin-cell/pin-cell.py

```diff
@@ -106,6 +106,11 @@
 solver = CPUSolver(geometry, track_generator)
 solver.setNumThreads(num_threads)
 solver.setSourceConvergenceThreshold(tolerance)
+
```

Comment:
Oops, I had that in to test your polar quadrature refactorization. I'll take it out.